### ISSUE TITLE
About rgb training 

### ISSUE THREAD

**Issue by jinliang20000** [cchen156/Learning-to-See-in-the-Dark#6]
paper has  sRGB input comparision,  I modify the last two level conv10(12 channel ) to output 3channel for the rgb  input image generated by rawpy.
the training loss is down to 0.025,   the result is not  bad, but some image is not real.   is this correct  way?

**Comment by cchen156** [comment]
The sRGB in the paper means the input is in sRGB space instead of RAW, not the output.

**Comment by jinliang20000** [comment]
Hi, my statement was wrong.  I mean the input is RGB image, not Raw.  thank you !

**Comment by cchen156** [comment]
What do you mean? How do you convert RAW to RGB and use it as input?

**Comment by jinliang20000** [comment]
I first transform your raw data(Sony) to rgb data by rawpy ,  and match the target brightness by  the ratio.  last,  change the input 4 channel of network to 3 for training,  also change  output channels(12 channels ) to 3 channels.

**Comment by cchen156** [comment]
The results should not be as good as ours, which we have tried. See the results on the test set, but not only the training loss.

**Comment by yueruchen** [comment]
@jinliang20000 
How long did it take to train? What's your final loss in train set?
Thanks,

**Comment by oneTaken** [comment]
This method seems interesting, can you report your result in the test set? @jinliang20000

**Comment by willswordh** [comment]
@yutaka329 Hi, do you mind share your result about your trained RGB model? If possible, can you also share the model weight or checkpoint? Thanks!
